paimon-core/src/main/java/org/apache/paimon/utils/BranchManager.java

```diff
@@ -142,6 +145,74 @@ public boolean fileExists(Path path) {
         }
     }
 
+    public void mergeBranch(String branchName) {
```

Comment:
And also I think we could provide a procedure for `mergeBranch`, WDYT?

Reply:
Of course, I intend to commit another pr to implement the procedure